pkg/v1/tkg/constants/clusterclass_mapping.go

```diff
@@ -279,7 +279,7 @@ const (
 
 	SPEC = "spec"
 
-	TopologyClassIncorrectValueErrMsg = "input cluster class file, attribute spec.topology.class has no value or incorrect value or not following correct naming convension"
+	TopologyClassIncorrectValueErrMsg = "input cluster class file, attribute spec.topology.class has no value or incorrect value or not following correct naming convention"
```

Comment:
I have fixed this typo as part of my PR as well. https://github.com/vmware-tanzu/tanzu-framework/pull/3053

Reply:
i have reverted my changes.